src/main.ts

```diff
@@ -4,7 +4,7 @@ import config from './config/environment';
 import moduleMap from './config/module-map';
 
 const resolverConfiguration: ResolverConfiguration = {
-  app: { rootName: config.modulePrefix },
+  app: { name: config.modulePrefix, rootName: config.modulePrefix },
```

Comment:
Ah yes, we need to derive `rootName` from `name` (if none is given) in glimmer-resolver. But this more explicit form is needed for now. 👍